jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/language/impl/JavaLanguage.kt

```diff
@@ -421,23 +433,51 @@ class JavaLanguage : TargetLanguage {
 
     private fun appendStartFunction(name: String, function: FunctionPresentation) {
         writeSeparated("public class ${classNameForStaticFunction(name)}")
-        inScope {
+        inScope({
             tabulate()
             writeVisibility(function.visibility)
             writeSeparated("static")
             writeTypeUsage(function.returnType)
             write(function.shortName)
             writeParametersList(function)
-            inScope {
+            inScope({
```

Comment:
you dont need to add parenthesis around lambdas as with named parameter argument kotlin will deduce that with lamba is the last in available parameters

Reply:
In this case, the lambda is followed by a Boolean variable, which is responsible for whether we need a semicolon. Therefore, these two arguments are wrapped in parentheses.